<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LasNewEntryDialog</class>
 <widget class="QDialog" name="LasNewEntryDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>912</width>
    <height>587</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Enter New Item</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>886</width>
        <height>521</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="barcodeLayout">
         <item>
          <widget class="QLabel" name="barcodeLabel">
           <property name="text">
            <string>Barcode: </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="barcodeValueLabel">
           <property name="text">
            <string>Barcode Value</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="nameLayout">
         <item>
          <widget class="QLabel" name="nameLabel">
           <property name="text">
            <string>Name: </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="nameEdit">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="descriptionLayout">
         <item>
          <widget class="QLabel" name="descriptionLabel">
           <property name="text">
            <string>Description: </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="descriptionEdit">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="quantityLayout">
         <item>
          <widget class="QLabel" name="quantityLabel">
           <property name="text">
            <string>Quantity: </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="quantitySpinBox">
           <property name="value">
            <number>1</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="locationLayout">
         <item>
          <widget class="QLabel" name="locationLabel">
           <property name="text">
            <string>Location: </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="locationEdit">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>LasNewEntryDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>LasNewEntryDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
